dev-util/hxtools/hxtools-20180301.recipe

```diff
@@ -96,11 +102,15 @@ PROVIDES="
 	cmd:git_track$secondaryArchSuffix
```

Comment:
I think we should use `commandSuffix` for this port. On `x86_gcc2` these commands will not work as expected. They're git plugins, which means they are usually called like this:

    git track # yes, no dash

with `secondaryArchSuffix`, it would be:

    git track-x86

which certainly no one expects.